Cargo.toml

```diff
@@ -113,7 +112,6 @@ include = [
     "include/ring-core/asm_base.h",
     "include/ring-core/base.h",
     "include/ring-core/check.h",
-    "include/ring-core/mem.h",
```

Comment:
We should be able to remove mem.h with this PR though, if we remove the include from mem.c.